modules/fedimint-ln-server/src/lib.rs

```diff
@@ -799,6 +799,12 @@ impl ServerModule for Lightning {
                         .ok_or_else(|| ApiError::not_found(String::from("Contract not found")))
                 }
             },
+            api_endpoint! {
+                "/clock_time",
+                async |module: &LightningModule, dbtx, _params: ()| -> SystemTime {
+                   Ok(module.consensus_clock_time(&mut dbtx).await.unwrap_or(UNIX_EPOCH))
```

Comment:
I'd rather return an `Option` here or have the `UNIX_EPOCH` be the default in all cases (i.e. happens in `consensus_clock_time`).